src/blocks/blocks.scss

```
@@ -0,0 +1,37 @@
+.blocks {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill, minmax($cellSize, 1fr));
+
+  &__block {
+    height: $cellSize;
+    box-sizing: border-box;
+
+    font-size: 100px;
+    line-height: $cellSize;
+    color: #fff;
+    text-align: center;
+
+    $cellColors:
```

It's not JavaScript, so use `-` to separate several words
```suggestion
    $cell-сolors:
```